File: core/src/androidTest/mock/com/mobilesolutionworks/android/controller/test/nested/RootWorksFragment.java
//package com.mobilesolutionworks.android.controller.test.nested;
//
//import android.os.Bundle;
//import android.support.annotation.Nullable;
//import android.view.View;
//import android.widget.TextView;
//
//import com.mobilesolutionworks.android.controller.test.R;
//
///**
// * Created by yunarta on 9/3/17.
// */
//
//public class RootWorksFragment extends EmptyWorksFragment {
//
//    @Override
//    public void onCreate(@Nullable Bundle savedInstanceState) {
//        super.onCreate(savedInstanceState);
//        if (savedInstanceState == null) {
//            getChildFragmentManager().beginTransaction()
//                    .add(new ChildWorksFragment(), "child")
//                    .commit();
//        }
//    }
//
//    @Override
//    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
//        super.onViewCreated(view, savedInstanceState);
//
//        TextView id = (TextView) view.findViewById(R.id.textView);
//        id.setText("child");
//    }
//
//}

File: test-app/src/main/kotlin/com/mobilesolutionworks/android/controller/test/nested/RootWorksFragment.kt
package com.mobilesolutionworks.android.controller.test.nested

import android.os.Bundle
import android.view.View
import android.widget.TextView

import com.mobilesolutionworks.android.controller.test.R

/**
 * Created by yunarta on 9/3/17.
 */

class RootWorksFragment : EmptyWorksFragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (savedInstanceState == null) {
            childFragmentManager.beginTransaction()
                  .add(ChildWorksFragment(), "child")
                  .commit()
        }
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        val id = view.findViewById(R.id.textView) as TextView
        id.text = "child"
    }

}
